src/pages/Rockets.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Rocket from '../components/Rocket';
+import {
+  selectAllRockets, fetchRockets,
+} from '../redux/rockets/rocketSlice';
+
+let didInitialRender = false;
+const Rockets = () => {
+  const rockets = useSelector(selectAllRockets);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (!didInitialRender) {
+      dispatch(fetchRockets());
+      didInitialRender = true;
+    }
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Your project looks great. Kindly avoid disabling linter checks in line. They help to maintain coding standards with your team and most of the time enforce best coding practices.

Make this change and your project will be ready for approval.